app/javascript/actions/building_type.js

```diff
@@ -0,0 +1,10 @@
+import {
+  ADD_BUILDING_TYPE,
+} from '../constants';
+
+export function addBuildingType(buildingType) {
```

Comment:
Can we call the param `buildingTypeName` instead of `buildingType` to make it clear it's just a string?

Reply:
Ah this was changed because it returns an actual `BuildingType` object instead of just the name. In the `addBuildingType` reducer, the state is updated with the entire object, rather than piecing the `name`, `questions`, `categories` and `id` together. Should I change that? I'm not sure what the convention is

Reply:
Hm on second thought, it's fine to leave it this way - most of our other reducers do the same thing and it makes sense because we want to put all of the response data into the store, not just the name